### PR TITLE
[WIP][DOC] 0.7.0 release note

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Add 0.7.0 release note

### What type of PR is it?
Documentation

### Todos
* [x] Confirm release date and update list of people who verified release once vote pass.
* [x] Update doap.rdf

### How should this be tested?
Open [zeppelin-release-0.7.0.md](https://github.com/minahlee/zeppelin/blob/28dfd26fa6f6fe87cd629770674dcbba10b50e04/releases/zeppelin-release-0.7.0.md) and see if there is missing feature that needs to be mentioned in release note.

### Screenshots (if appropriate)
Download page
screen shot 2017-02-04 at 10 48 07 pm

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? -

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Looks good to me!

File: download.md [hunk 1]
```diff
@@ -21,33 +21,27 @@ limitations under the License.
...
     * Source:
-    <a style="cursor:pointer" onclick="ga('send', 'event', 'download', 'zeppelin-src', '0.6.2'); window.location.href='http://www.apache.org/dyn/closer.cgi/zeppelin/zeppelin-0.6.2/zeppelin-0.6.2.tgz'">zeppelin-0.6.2.tgz</a> (1.4 MB,
-    [pgp](https://www.apache.org/dist/zeppelin/zeppelin-0.6.2/zeppelin-0.6.2.tgz.asc),
-    [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.6.2/zeppelin-0.6.2.tgz.md5),
-    [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.6.2/zeppelin-0.6.2.tgz.sha512))
-
-    <blockquote style="margin-top: 10px;">
-      <p><strong>Note</strong>: From Zeppelin version 0.6.2, Spark interpreter in binary package is compatible with Spark 2.0 & Scala 2.11 and Spark 1.6(or previous) & Scala 2.10. You can use even different version of Spark at the same time if you set different SPARK_HOME in interpreter setting page.</p>
-    </blockquote>
+    <a style="cursor:pointer" onclick="ga('send', 'event', 'download', 'zeppelin-src', '0.7.0'); window.location.href='http://www.apache.org/dyn/closer.cgi/zeppelin/zeppelin-0.7.0/zeppelin-0.7.0.tgz'">zeppelin-0.7.0.tgz</a> (1.9 MB,
```
Comment (felixcheung) [inline download.md]: should we perhaps add a link to "what's new" page here?

File: supported_interpreters.md [hunk 2]
```diff
@@ -31,15 +31,18 @@ Please check the below table before you download Zeppelin package.
...
     <th style="width:22%">0.6.0</th>
     <th style="width:22%">0.5.6</th>
-    <th style="width:22%">0.5.5</th>
   </tr>
   <tr>
     <td rowspan="2">
       <h6><a href="http://spark.apache.org/" target="_blank">Spark</a><h6>
     </td>
+    <td rowspan="2">
+        1.1.x, 1.2.x, 1.3.x 1.4.x, 1.5.x, 1.6.x, 2.0.x <strong>2.1.0</strong>
```
Comment (felixcheung) [inline supported_interpreters.md]: I thought we took out 1.1, 1.2, 1.3 from tests?

**Comment by felixcheung** [comment]
great thanks!

**Comment by minahlee** [comment]
Merging it to deploy 0.7.0 document.

**Pull request closed without merging**